pages/home/script.js

```diff
@@ -143,6 +146,8 @@ window.onload = async function () {
         category: hazardDetail.category.name,
         hazard: hazardDetail.option.name,
         location: hazardDetail.location.address,
```

Comment:
can we mirror the know structure 

location {
  lat
  lng
  address
}

instead of having these attributes separate?

Reply:
I just didn't do that change because this `class` has been used for other features